routes/api.php

```diff
@@ -20,3 +20,9 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::post('/register', [RegisterController::class, 'registerUser']);
```

Comment:
There's no need to redefine the `/register` endpoint again, it's already defined in `routes/auth.php`.